src/codegen/generate.ts

```diff
@@ -30,6 +30,11 @@ export const contentTypes: Record<string, ContentType> = {
   "multipart/form-data": "multipart",
 };
 
+// augment SchemaObject type to allow slowly adopting new OAS3.1+ features
+type SchemaObject = OpenAPIV3.SchemaObject & {
+  const?: any;
```

Comment:
Not a big fan of `any`. I would very much prefer an `unknown` instead.
Should be an easy change since you're already doing `typeof` checks where it's used.

Reply:
The decision to use "any" was because the `enum` prop is currently typed as "any[]", and since this is meant to be a shorthand for a single-value enum I didn't want to introduce any potential type discrepancies. Either way will compile fine and neither provide much protection in the current code, so I'm happy to change it.